webAO/master.js

```diff
@@ -50,7 +50,7 @@ function onOpen(_e) {
 	masterserver.send("VC#%");
 }
 
-function checkOnline(serverID, coIP) {
```

Comment:
you don't need to define this as async because all it does is set up callbacks, there's nothing actually blocking here